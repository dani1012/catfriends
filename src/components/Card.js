import React from 'react';

const Card = ({name,email,id})=> {

	return (

    <div className = 'bg-light-green br2 dib pa3 ma2 grow bw2 shadow-5'>
	     <img alt = 'robots' src = {`https://robohash.org/set_set4/${id}/?size=200x200`}/>
	     <div>
	     	<h4>{name}</h4>
	     	<p>{email}</p>
	     </div>
    </div>	

		);

}

export default Card;