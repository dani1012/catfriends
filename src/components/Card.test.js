import React from 'react';
import Card from './Card';
import Enzyme from 'enzyme';
import {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

Enzyme.configure({ adapter: new Adapter() });



it ('expects to render Card component', () => {
	const wrapper = shallow(<Card/>);
	expect(toJson(wrapper)).toMatchSnapshot();
})


it ('expects to render Card component', () => {
	const wrapper = shallow(<Card/>);
	expect((wrapper).length).toEqual(1);
})