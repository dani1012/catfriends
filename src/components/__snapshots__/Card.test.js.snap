// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render Card component 1`] = `
<div
  className="tc bg-light-green br2 dib pa3 ma2 grow bw2 shadow-5"
>
  <img
    alt="robots"
    src="https://robohash.org/set_set4/undefined/?size=200x200"
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
